spec/imsc-hrm.html

```diff
@@ -616,35 +616,35 @@ <h2>Algorithm</h2>
     <section id='paint-regions'>
       <h2>Paint Regions</h2>
 
-      <p>The total normalized drawing area S(E<sub>n</sub>) for <a>Intermediate Synchronic Document</a> E<sub>n</sub> is given by:</p>
+      <p>The total normalized drawing area <dfn>S</dfn>(E<sub>n</sub>) for <a>Intermediate Synchronic Document</a> E<sub>n</sub> is given by:</p>
 
-      <p class="equation">S(E<sub>n</sub>) = CLEAR(E<sub>n</sub>) + PAINT(E<sub>n</sub> )</p>
+      <p class="equation">S(E<sub>n</sub>) = <a>CLEAR</a>(E<sub>n</sub>) + <a>PAINT</a>(E<sub>n</sub> )</p>
 
-      <p>where CLEAR(E<sub>n</sub>) = 1.</p>
+      <p>where <dfn>CLEAR</dfn>(E<sub>n</sub>) = 1.</p>
 
       <p class='note'>To ensure consistency of the Back Buffer, a new <a>Intermediate Synchronic Document</a> requires
       clearing of the <a>Root Container Region</a>.</p>
 
-      <p>PAINT(E<sub>n</sub>) is the normalized area to be painted for all regions that are used in <a>Intermediate
+      <p><dfn>PAINT</dfn>(E<sub>n</sub>) is the normalized area to be painted for all regions that are used in <a>Intermediate
       Synchronic Document</a> E<sub>n</sub> according to:</p>
 
-      <p class="equation">PAINT(E<sub>n</sub>) = ∑<sub>R<sub>i</sub>∈R<sub>p</sub></sub> NSIZE(R<sub>i</sub>) ∙
-      NBG(R<sub>i</sub>)</p>
+      <p class="equation">PAINT(E<sub>n</sub>) = ∑<sub>R<sub>i</sub>∈R<sub>p</sub></sub> <a>NSIZE</a>(R<sub>i</sub>) ∙
```

Comment:
```suggestion
      <p class="equation"><a>PAINT</a>(E<sub>n</sub>) = ∑<sub>R<sub>i</sub>∈R<sub>p</sub></sub> <a>NSIZE</a>(R<sub>i</sub>) ∙
```

Reply:
I thought about this too and decided it'd be better _not_ to do this, because the defined term is just above. I'm persuadable though - presumably for consistency you'd want the same change made for all the equations whose left side is a defined term?

Reply:
I am ok to not link terms in equations. Just wanted to make sure you had not missed it by mistake.